src/domains/checkout-app/04_apim_checkout_authentication.tf

```diff
@@ -26,7 +26,7 @@ module "apim_checkout_authentication" {
 }
 
 resource "azurerm_api_management_api_version_set" "checkout_auth_service_api_v1" {
-  name                = "${local.parent_project}-auth-service-api"
+  name                = "${var.prefix}-${var.env_short}-${var.domain}-auth-service-api"
```

Comment:
Could you use the [local project value](https://github.com/pagopa/pagopa-infra/blob/396f5062ddea6ab9d70e2d7ac7ac0f7c1d3703d4/src/domains/checkout-app/99_locals.tf#L2) here, like it's done for other domains such eCommerce?